tests/run_tests.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+function show_help_and_exit()
+{
+    echo "Usage ${SCRIPT} [options]"
+    echo "    options with (*) must be provided"
+    echo "    -h -?          : get this help"
+    echo "    -c             : specify test cases to execute (default: none, executed all matched)"
+    echo "    -d             : specify DUT name (*)"
+    echo "    -e             : specify extra parameter(s) (default: none)"
+    echo "    -f             : specify testbed file (default testbed.csv)"
+    echo "    -i             : specify inventory name"
+    echo "    -k             : specify file log level: error|warning|info|debug (default debug)"
+    echo "    -l             : specify cli log level: error|warning|info|debug (default warning)"
+    echo "    -m             : specify test method group|individual|dump (default group)"
+    echo "    -o             : omit the file logs"
+    echo "    -n             : specify testbed name (*)"
+    echo "    -s             : specify list of scripts to skip (default: none)"
+    echo "    -t             : specify toplogy: t0|t1|any|combo like t0,any (*)"
+    echo "    -u             : bypass util group"
+
+    exit $1
+}
```

Comment:
This is a pretty good help message, but I think it would be helpful to include a README with some usage examples and such.

Reply:
I agree. let's come back with another PR.